api/src/main/java/jakarta/xml/bind/ServiceLoaderUtil.java

```diff
@@ -74,6 +76,25 @@ static Object lookupUsingOSGiServiceLoader(String factoryId, Logger logger) {
         }
     }
 
+    @SuppressWarnings("unchecked")
+    static Iterable<Object> lookupsUsingOSGiServiceLoader(String factoryId, Logger logger) {
```

Comment:
use `<T> Iterable<T>` so the compiler can do its work and figure out the right type itself when appropriate

Reply:
I like doing `Iterable<T>`. I used `Object` there to stay as close to the other method which returns just Object and not `T`. I think if I did `<T> Iterable<T>` someone would have given feedback to use Object instead.

Reply:
see [similar class](https://github.com/eclipse-ee4j/jax-ws-api/blob/master/api/src/main/java/jakarta/xml/ws/spi/ServiceLoaderUtil.java#L69) - there is no Object and the right type is checked at the compile time by the compiler itself

Reply:
I'm a big fan of that paradigm, and use it a lot elsewhere too. It's just that `lookupUsingOSGiServiceLoader` returned `Object`.